extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -123,13 +148,32 @@ function SidebarWithRef(
     [open, close, toggle]
   );
 
+  useLayoutEffect(() => {
+    if (side) {
+      return;
+    }
+    const sidebarElement = sidebarRef.current;
+    if (!sidebarElement) {
+      return;
+    }
+    setSide(calculateSide(sideProp, sidebarElement.ownerDocument));
+  }, [sideProp, side, mounted]);
```

Comment:
Do we need `mounted` here?

Reply:
Yes, because on the initial render, `mounted = false` so we don't get `sidebarRef` and end up exiting early without doing the `setSide` calculation. :/

Reply:
Got it.

Reply:
Though it's probably worth commenting this fact.

Reply:
I'm not convinced we should conditionally render `sideBarRef` with `mounted && sidebarRef` if we need to work around `sidebarRef` only sometimes being defined. We now also add hidden styles for sidebar and mask before open anyway, so the conditional render is not even getting us out of doing that.

In the future we will also likely want to always render the content if we want sidebar content to be matchable with `content-visibility: hidden-matchable`.

Reply:
I agree this is definitely worth exploring!